src/test/java/hudson/plugins/active_directory/ActiveDirectoryDomainIntegrationTest.java

```diff
@@ -0,0 +1,171 @@
+package hudson.plugins.active_directory;
+
+import org.htmlunit.FailingHttpStatusCodeException;
+import org.htmlunit.html.HtmlButton;
+import org.htmlunit.html.HtmlElement;
+import org.htmlunit.html.HtmlForm;
+import org.htmlunit.html.HtmlInput;
+import org.htmlunit.html.HtmlPage;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.jvnet.hudson.test.FlagRule;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.recipes.LocalData;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assume.assumeFalse;
+
+import static hudson.Functions.isWindows;
+
+public class ActiveDirectoryDomainIntegrationTest {
```

Comment:
These tests are explicitly for FiPS so it should be noted in the class name for easier future investigations.